codility_training/lessons.lesson05.PrefixSums.PassingCars/sangmandu.py

```diff
@@ -0,0 +1,21 @@
+# you can write to stdout for debugging purposes, e.g.
+# print("this is a debug message")
+
+def solution(A):
+    # write your code in Python 3.6
+    pass
+    zero_cnt = A.count(0)
+    one_cnt = len(A)- zero_cnt
+    cnt = 0
+    for i in A:
+        if i == 0:
+            cnt += one_cnt

```

Comment:
```suggestion
            cnt += one_cnt
            if (cnt > 1000000000) return -1
```
이 부분에 추가하는게 좋을 듯 합니다~

Reply:
> return은 이 부분에 추가하는게 좋을 듯 합니다~

1000000000 전까지는 매번 if조건을 검사해야 할 거 같은데, 1000000000이후 값은 검사 하지 않아도 되니까 그렇게 하신건가요?

Reply:
맞습니다! 항상 이렇게 해야한다라기보단 상황에 따라 적절하게 대처하면 좋을 것 같아요~